src/pages/contacts-page/contacts-page.js

```diff
@@ -62,16 +55,12 @@ const ContactsPage = () => {
   useEffect(() => {
     dispatch(
       getContacts({
-        limit: contactsPerPage,
-        skip: contactsCurrentPage * contactsPerPage,
-        contactsPerPage
+        limit: rowsPerPage,
+        skip: currentPage * rowsPerPage,
+        rowsPerPage
       })
     );
-  }, [dispatch, contactsPerPage, contactsCurrentPage]);
-
-  const changePageHandler = (e, pageIndex) => {
-    dispatch(setContactsCurrentPage(pageIndex));
-  };
+  }, [dispatch, rowsPerPage, currentPage]);
 
   const contactItems = contacts
     ? contacts.map((contact, index) => (
```

Comment:
Please change to lodash function `map` https://lodash.com/docs/4.17.15#map.

Reply:
Fixed all